hard_switch_control:
  ros__parameters:
    tolerance_distance: 0.1
    angular_kp: 1.5
    linear_kp: 0.6
    linear_acc: 0.025
    angular_acc: 0.2
    avoiding_d: 1.5
    stop_d: 0.4
    lidar_vel: 0.5
    lidar_turning_kp: 0.4
    epsilon: 0.1